src/app/modules/user/user.controller.ts

```diff
@@ -22,8 +23,8 @@ const logger = createLoggerWithLabel(module)
  * @returns 400 on OTP creation or SMS send failure
  */
 export const handleContactSendOtp: RequestHandler<
-  {},
-  {},
+  Record<string, string>,
+  string,
   { contact: string; userId: string }
 > = async (req, res) => {
```

Comment:
why does `handleSns` need a return type but this doesn't?

Reply:
It's already pre-typed by `RequestHandler`, but good point I will standardize this function too